CONTRIBUTING.md

```diff
@@ -119,16 +119,23 @@ Before you submit your pull request consider the following guidelines:
 
 That's it! Thank you for your contribution!
 
-#### <a name="testing"></a> Testing approach
+#### <a name="testing"></a> Testing 
 
-Use the [BDD style](http://guide.agilealliance.org/guide/bdd.html) of testing.
-Tests should be organised around scenarios rather than rigidly around class
-structure and method names, for example "Creating an application" rather than
+Test are organised around scenarios ([BDD style](http://guide.agilealliance.org/guide/bdd.html)) and/or single units ([Unit Tests](http://guide.agilealliance.org/guide/unittest.html)) of work. In the best case we have unit test to verify that every single unit is working as expected and scenarios to ensure that they all play well together.
```

Comment:
Tests are organised following two different approaches: [BDD style](http://guide.agilealliance.org/guide/bdd.html) and  [Unit Tests](http://guide.agilealliance.org/guide/unittest.html). Our aim is to write unit tests that verify every single unit is working as expected in isolation, as well as creating scenarios that ensure all the moving parts play well together.